src/Shared/Microsoft.DncEng.CommandLineLib/Authentication/Constants.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.DncEng.CommandLineLib.Authentication
+{
+    public static class Constants
```

Comment:
Generic "constants" classes always get a little weird.

It seems like right now the only class that actually uses the tenantId is the AzureServiceTokenProviderCredential class. Could it be a member of that class?

Reply:
I will move it.